internal/pkg/manifest/storage.go

```diff
@@ -11,8 +11,7 @@ import (
 )
 
 var (
-	errUnmarshalEFSOpts        = errors.New(`cannot unmarshal efs field into bool or map`)
-	errInvalidEFSConfiguration = errors.New(`must specify one, not both, of "uid/gid" and "id/root_dir/auth"`)
+	errUnmarshalEFSOpts = errors.New(`cannot unmarshal efs field into bool or map`)
```

Comment:
```suggestion
	errUnmarshalEFSOpts = errors.New(`cannot unmarshal "efs" field into bool or map`)
```

It seems from `manifes/errors.go` that we add quotations around fields names.